src/components/Sponsors/Sponsors.js

```diff
@@ -9,7 +9,7 @@ export function Sponsors () {
   return (
     <Fragment>
       <FullWidthContainer className={styles.sponsors__hero}>
-        <h3 className={styles.sponsors__title}>Sponsors 2020</h3>
+        <h3 className={styles.sponsors__title}>Sponsors 2021</h3>
```

Comment:
Well, it's not that simple regarding sponsoring.
Let's just remove the year.
```suggestion
        <h3 className={styles.sponsors__title}>Sponsors</h3>
```